src/omnicore/tx.cpp

```diff
@@ -1678,6 +1785,7 @@ int CMPTransaction::logicMath_CreatePropertyManaged()
         return (PKT_ERROR_SP -21);
     }
 
+    // TODO: When we think about making unique tokens live we need to activate, and reusing current version txs may be problematic
```

Comment:
I was wondering about this: right now it would allow creating UIT tokens even without activation, as far as I can see.